charts/airbyte-pod-sweeper/Chart.lock

```diff
@@ -1,6 +0,0 @@
-dependencies:
```

Comment:
why remove this?

Reply:
To get rid of the bitnami security warnings?

Reply:
Leftovers from testing the charts

Reply:
What do you mean?